CMakeLists.txt

```diff
@@ -1127,6 +1127,10 @@ else()
   message(STATUS "Building headless")
 endif()
 
+if (WIN32)
+  list(APPEND USED_SOURCES res/furnace.rc)
```

Comment:
Wasn't this added somewhere else?

Reply:
Yeah, I realized that after I built... if it still works you may remove this line because I'm a dummy